mantidproject/mantid
# Add sub groups to fit sequential output in Muon Analysis

This issue covers a reorganisation of the output of Muon Analysis fit sequential functionality.
The output workspaces should now be organised into sub group workspaces.

**To test:**

* Open the Muon Analysis interface (Interfaces > Muon > Muon Analysis)
* Select the instrument as `EMU` from the drop down menu
* Use input of `20717` as the run number in the `Load run` box (the run should load automatically)
* Select the `Data Analysis` tab from the top of the interface
* Right click the FitFunction property tree and select `Add function`
* Select any function
* Click the `Fit` drop down menu and select `Sequential fit`
* Add `20` to runs field so it reads `20717-20`
* Click start 
* Close the Sequential Fitting dialog
* Expand the MounSeqFit_Label workspace group
  * Ensure this is populated by 4 workspace groups
  * Each of these groups should consist of a:
     * `..._NormalisedCovarianceMatrix`
     * `..._Parameters`
     * `..._Workspace` 

Fixes #21826

**Release Notes** 
*Small change - no need for release notes*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
